service_info/templates/cms/base.html

```diff
@@ -70,7 +75,7 @@
         {% endblock content %}
       {% endblock wrap-content %}
       {% if request.current_page.ratingextension.include_rating %}
-        <section class="row rating">
+        <section class="row rating{% block rating-class %}{% endblock rating-class %}">
```

Comment:
more foolproof to put the space here outside of the {% block %} instead of requiring each override to have an extra space?